src/TideNewsOperation.php

```diff
@@ -0,0 +1,107 @@
+<?php
+
+namespace Drupal\tide_news;
+
+use Drupal\user\Entity\Role;
+use Drupal\workflows\Entity\Workflow;
+
+/**
+ * Helper class for install/update ops.
+ */
+class TideNewsOperation {
+
+  /**
+   * Enable editorial workflow and shceduled transitions.
+   */
+  public static function enableNecessaryModules() {
+    // Enable Editorial workflow if workflow module is enabled.
+    $moduleHandler = \Drupal::service('module_handler');
+    if ($moduleHandler->moduleExists('workflows')) {
```

Comment:
Let's get in the habit of early return on these module present checks. No need to nest large operations within a module check, just exit early.

This also means I would put the work for each in seperate functions (lines 17 to 25 in one function, lines 28 to 46 in another function) so you can perform early return on both module checks properly